ts/features/walletV3/payment/screens/WalletPaymentInputFiscalCodeScreen.tsx

```diff
@@ -0,0 +1,146 @@
+import {
+  Body,
+  ButtonSolid,
+  ContentWrapper,
+  H2,
+  IOStyles,
+  TextInputValidation,
+  VSpacer
+} from "@pagopa/io-app-design-system";
+import {
+  AmountInEuroCents,
+  PaymentNoticeNumberFromString,
+  RptId
+} from "@pagopa/io-pagopa-commons/lib/pagopa";
+import { OrganizationFiscalCode } from "@pagopa/ts-commons/lib/strings";
+import { RouteProp, useNavigation, useRoute } from "@react-navigation/native";
+import { sequenceS } from "fp-ts/lib/Apply";
+import * as O from "fp-ts/lib/Option";
+import { flow, pipe } from "fp-ts/lib/function";
+import React from "react";
+import {
+  KeyboardAvoidingView,
+  Platform,
+  SafeAreaView,
+  View
+} from "react-native";
+import BaseScreenComponent from "../../../../components/screens/BaseScreenComponent";
+import I18n from "../../../../i18n";
+import {
+  AppParamsList,
+  IOStackNavigationProp
+} from "../../../../navigation/params/AppParamsList";
+import ROUTES from "../../../../navigation/routes";
+import { paymentInitializeState } from "../../../../store/actions/wallet/payment";
+import { useIODispatch } from "../../../../store/hooks";
+import themeVariables from "../../../../theme/variables";
+import { emptyContextualHelp } from "../../../../utils/emptyContextualHelp";
+import {
+  decodeOrganizationFiscalCode,
+  validateOrganizationFiscalCode
+} from "../../common/utils/validation";
+import { WalletPaymentParamsList } from "../navigation/params";
+
+export type WalletPaymentInputFiscalCodeScreenRouteParams = {
+  paymentNoticeNumber: O.Option<PaymentNoticeNumberFromString>;
+};
+
+type WalletPaymentInputFiscalCodeRouteProps = RouteProp<
+  WalletPaymentParamsList,
+  "WALLET_PAYMENT_INPUT_FISCAL_CODE"
+>;
+
+type InputState = {
+  fiscalCodeText: string;
+  fiscalCode: O.Option<OrganizationFiscalCode>;
+};
+
+const WalletPaymentInputFiscalCodeScreen = () => {
+  const { params } = useRoute<WalletPaymentInputFiscalCodeRouteProps>();
+  const dispatch = useIODispatch();
+  const navigation = useNavigation<IOStackNavigationProp<AppParamsList>>();
+  const [inputState, setInputState] = React.useState<InputState>({
+    fiscalCodeText: "",
+    fiscalCode: O.none
+  });
+
+  const navigateToTransactionSummary = () => {
+    pipe(
+      sequenceS(O.Monad)({
+        paymentNoticeNumber: params.paymentNoticeNumber,
+        organizationFiscalCode: inputState.fiscalCode
+      }),
+      O.chain(flow(RptId.decode, O.fromEither)),
+      O.map(rptId => {
+        dispatch(paymentInitializeState());
+        navigation.navigate(ROUTES.WALLET_NAVIGATOR, {
+          screen: ROUTES.PAYMENT_TRANSACTION_SUMMARY,
+          params: {
+            // FIXME: Set the initial amount to a fixed value (1) because it is not used, waiting to be removed from the API
+            initialAmount: "1" as AmountInEuroCents,
```

Comment:
What do you think to provide also a jira ticket link to this fixme comment?

Reply:
Since there is no evidence that this will be ever removed from the API and since this is a placeholder, I removed the FIXME comment in https://github.com/pagopa/io-app/pull/5130/commits/5502b7301d2ef318407ba56739518c10fb64fce1